Patch from
https://bugs.launchpad.net/ubuntu/+source/xkeyboard-config/+bug/93077 to
avoid the use of "any" so that console-setup is happier.

only in patch2:
unchanged:
--- xkeyboard-config-1.0~cvs.20070916-1ubuntu1.orig/symbols/cs	2007-11-21 19:18:13.000000000 +0100
+++ xkeyboard-config-1.0~cvs.20070916-1ubuntu1/symbols/cs	2007-11-23 12:16:50.000000000 +0100
@@ -150,56 +150,56 @@
 
 partial hidden alphanumeric_keys
 xkb_symbols "level3" {
-  key <TLDE> {   [ any,any,     notsign,             notsign     ]   }; // ` ~
-  key <AE01> {   [ any,any,     dead_tilde,          asciitilde  ]   }; // 1 !
-  key <AE02> {   [ any,any,     dead_caron,          caron 	 ]   }; // 2 @
-  key <AE03> {   [ any,any,     dead_circumflex,     asciicircum ]   }; // 3 #
-  key <AE04> {   [ any,any,     dead_breve,          breve 	 ]   }; // 4 $
-  key <AE05> {   [ any,any,     dead_abovering,      degree	 ]   }; // 5 %
-  key <AE06> {   [ any,any,     dead_ogonek,         ogonek	 ]   }; // 6 ^
-  key <AE07> {   [ any,any,     dead_grave,          grave 	 ]   }; // 7 &
-  key <AE08> {   [ any,any,     dead_abovedot,       abovedot 	 ]   }; // 8 *
-  key <AE09> {   [ any,any,     dead_acute,          apostrophe  ]   }; // 9 (
-  key <AE10> {   [ any,any,     dead_doubleacute,    doubleacute ]   }; // 0 )
-  key <AE11> {   [ any,any,     dead_diaeresis,      diaeresis 	 ]   }; // - _
-  key <AE12> {   [ any,any,     dead_cedilla,        cedilla 	 ]   }; // = +
-
-  key <AD01> {   [ any,any,     backslash,           Greek_OMEGA ]   }; // q
-  key <AD02> {   [ any,any,     bar,                 Lstroke 	 ]   }; // w
-  key <AD03> {   [ any,any,     EuroSign,            EuroSign 	 ]   }; // e
-  key <AD04> {   [ any,any,     paragraph,           registered  ]   }; // r
-  key <AD05> {   [ any,any,     tslash,              Tslash 	 ]   }; // t // ALPHABETIC
-  key <AD06> {   [ any,any,     leftarrow,           yen 	 ]   }; // y
-  key <AD07> {   [ any,any,     downarrow,           uparrow 	 ]   }; // u
-  key <AD08> {   [ any,any,     rightarrow,          idotless 	 ]   }; // i
-  key <AD09> {   [ any,any,     oslash,              Ooblique 	 ]   }; // o // ALPHABETIC
-  key <AD10> {   [ any,any,     thorn,               THORN 	 ]   }; // p // ALPHABETIC
-  key <AD11> {   [ any,any,     division,            dead_abovering ]   }; // [
-  key <AD12> {   [ any,any,     multiply,            dead_macron ]   }; // ]
-
-  key <AC01> {   [ any,any,     ae,            	     AE               ]   }; // a // ALPHABETIC
-  key <AC02> {   [ any,any,     doublelowquotemark,  guillemotright   ]   }; // s
-  key <AC03> {   [ any,any,     leftdoublequotemark, guillemotleft    ]   }; // d
-  key <AC04> {   [ any,any,     bracketleft,         ordfeminine      ]   }; // f
-  key <AC05> {   [ any,any,     bracketright,        ENG 	      ]   }; // g
-  key <AC06> {   [ any,any,     hstroke,             Hstroke 	      ]   }; // h // ALPHABETIC
-  key <AC07> {   [ any,any,     NoSymbol,            NoSymbol 	      ]   }; // j
-  key <AC08> {   [ any,any,     lstroke,             ampersand 	      ]   }; // k
-  key <AC09> {   [ any,any,     lstroke,             Lstroke 	      ]   }; // l
-  key <AC10> {   [ any,any,     dead_acute,          dead_doubleacute ]   }; // ;
-  key <AC11> {   [ any,any,     ssharp,              dead_caron       ]   }; // '
-  key <BKSL> {   [ any,any,     currency,            dead_breve       ]   }; // \
-
-  key <AB01> {   [ any,any,     leftsinglequotemark, guillemotright   ]   }; // z
-  key <AB02> {   [ any,any,     rightsinglequotemark,guillemotleft    ]   }; // x
-  key <AB03> {   [ any,any,     cent,                copyright 	      ]   }; // c
-  key <AB04> {   [ any,any,     at,                  grave 	      ]   }; // v
-  key <AB05> {   [ any,any,     braceleft,           apostrophe       ]   }; // b
-  key <AB06> {   [ any,any,     braceright,          braceright       ]   }; // n
-  key <AB07> {   [ any,any,     asciicircum,         masculine 	      ]   }; // m
-  key <AB08> {   [ any,any,     less,                multiply 	      ]   }; // , <
-  key <AB09> {   [ any,any,     greater,             division 	      ]   }; // . >
-  key <AB10> {   [ any,any,     emdash,              endash   	      ]   }; // / ?
+  key <TLDE> {   [ grave,asciitilde,     notsign,             notsign     ]   }; // ` ~
+  key <AE01> {   [ 1,exclam,     dead_tilde,          asciitilde  ]   }; // 1 !
+  key <AE02> {   [ 2,quotedbl,     dead_caron,          caron 	 ]   }; // 2 @
+  key <AE03> {   [ 3,numbersign,     dead_circumflex,     asciicircum ]   }; // 3 #
+  key <AE04> {   [ 4,dollar,     dead_breve,          breve 	 ]   }; // 4 $
+  key <AE05> {   [ 5,percent,     dead_abovering,      degree	 ]   }; // 5 %
+  key <AE06> {   [ 6,ampersand,     dead_ogonek,         ogonek	 ]   }; // 6 ^
+  key <AE07> {   [ 7,slash,     dead_grave,          grave 	 ]   }; // 7 &
+  key <AE08> {   [ 8,parenleft,     dead_abovedot,       abovedot 	 ]   }; // 8 *
+  key <AE09> {   [ 9,parenright,     dead_acute,          apostrophe  ]   }; // 9 (
+  key <AE10> {   [ 0,equal,     dead_doubleacute,    doubleacute ]   }; // 0 )
+  key <AE11> {   [ apostrophe,question,     dead_diaeresis,      diaeresis 	 ]   }; // - _
+  key <AE12> {   [ plus,asterisk,     dead_cedilla,        cedilla 	 ]   }; // = +
+
+  key <AD01> {   [ q,Q,     backslash,           Greek_OMEGA ]   }; // q
+  key <AD02> {   [ w,W,     bar,                 Lstroke 	 ]   }; // w
+  key <AD03> {   [ e,E,     EuroSign,            EuroSign 	 ]   }; // e
+  key <AD04> {   [ r,R,     paragraph,           registered  ]   }; // r
+  key <AD05> {   [ t,T,     tslash,              Tslash 	 ]   }; // t // ALPHABETIC
+  key <AD06> {   [ z,Z,     leftarrow,           yen 	 ]   }; // y
+  key <AD07> {   [ u,U,     downarrow,           uparrow 	 ]   }; // u
+  key <AD08> {   [ i,I,     rightarrow,          idotless 	 ]   }; // i
+  key <AD09> {   [ o,O,     oslash,              Ooblique 	 ]   }; // o // ALPHABETIC
+  key <AD10> {   [ p,P,     thorn,               THORN 	 ]   }; // p // ALPHABETIC
+  key <AD11> {   [ scaron,Scaron,     division,            dead_abovering ]   }; // [
+  key <AD12> {   [ dstroke,Dstroke,     multiply,            dead_macron ]   }; // ]
+
+  key <AC01> {   [ a,A,     ae,            	     AE               ]   }; // a // ALPHABETIC
+  key <AC02> {   [ s,S,     doublelowquotemark,  guillemotright   ]   }; // s
+  key <AC03> {   [ d,D,     leftdoublequotemark, guillemotleft    ]   }; // d
+  key <AC04> {   [ f,F,     bracketleft,         ordfeminine      ]   }; // f
+  key <AC05> {   [ g,G,     bracketright,        ENG 	      ]   }; // g
+  key <AC06> {   [ h,H,     hstroke,             Hstroke 	      ]   }; // h // ALPHABETIC
+  key <AC07> {   [ j,J,     NoSymbol,            NoSymbol 	      ]   }; // j
+  key <AC08> {   [ k,K,     lstroke,             ampersand 	      ]   }; // k
+  key <AC09> {   [ l,L,     lstroke,             Lstroke 	      ]   }; // l
+  key <AC10> {   [ ccaron,Ccaron,     dead_acute,          dead_doubleacute ]   }; // ;
+  key <AC11> {   [ cacute,Cacute,     ssharp,              dead_caron       ]   }; // '
+  key <BKSL> {   [ zcaron,Zcaron,     currency,            dead_breve       ]   }; // \
+
+  key <AB01> {   [ y,Y,     leftsinglequotemark, guillemotright   ]   }; // z
+  key <AB02> {   [ x,X,     rightsinglequotemark,guillemotleft    ]   }; // x
+  key <AB03> {   [ c,C,     cent,                copyright 	      ]   }; // c
+  key <AB04> {   [ v,V,     at,                  grave 	      ]   }; // v
+  key <AB05> {   [ b,B,     braceleft,           apostrophe       ]   }; // b
+  key <AB06> {   [ n,N,     braceright,          braceright       ]   }; // n
+  key <AB07> {   [ m,M,     asciicircum,         masculine 	      ]   }; // m
+  key <AB08> {   [ comma,semicolon,     less,                multiply 	      ]   }; // , <
+  key <AB09> {   [ period,colon,     greater,             division 	      ]   }; // . >
+  key <AB10> {   [ minus,underscore,     emdash,              endash   	      ]   }; // / ?
 
 };
 
--- xkeyboard-config-1.0~cvs.20070916-1ubuntu1.orig/symbols/hr	2007-11-21 19:18:13.000000000 +0100
+++ xkeyboard-config-1.0~cvs.20070916-1ubuntu1/symbols/hr	2007-11-23 12:16:50.000000000 +0100
@@ -23,28 +23,28 @@
 
     include "us"
 
-    key <AD01> { [    any,    any,	backslash	        ] };
-    key <AD02> { [    any,    any,	bar      	        ] };
-    key <AD03> { [    any,    any, 	EuroSign 	        ] };
-    key <AD06> { [    any,    any,	z,		Z       ] };
-    key <AD11> { [    any,    any, 	scaron,		Scaron  ] };
-    key <AD12> { [    any,    any, 	dstroke, 	Dstroke ] };
-    key <AC04> { [    any,    any,	bracketleft             ] };
-    key <AC05> { [    any,    any,	bracketright            ] };
-    key <AC08> { [    any,    any, 	lstroke                 ] };
-    key <AC09> { [    any,    any,	Lstroke                 ] };
-    key <AC10> { [    any,    any,	ccaron,		Ccaron  ] };
-    key <AC11> { [    any,    any, 	cacute,		Cacute  ] };
-    key <LSGT> { [    any,    any, 	bar			] };
-    key <AB01> { [    any,    any,	y,		Y       ] };
-    key <AB04> { [    any,    any,	at                      ] };
-    key <AB05> { [    any,    any,	braceleft               ] };
-    key <AB06> { [    any,    any,	braceright              ] };
-    key <AB07> { [    any,    any,	section                 ] };
-    key <AB08> { [    any,    any, 	semicolon               ] };
-    key <AB09> { [    any,    any, 	colon			] };
-    key <AB10> { [    any,    any, 	minus,    underscore  	] };
-    key <BKSL> { [    any,    any,	zcaron,	    Zcaron      ] };
+    key <AD01> { [    q,    Q,	backslash	        ] };
+    key <AD02> { [    w,    W,	bar      	        ] };
+    key <AD03> { [    e,    E, 	EuroSign 	        ] };
+    key <AD06> { [    y,    Y,	z,		Z       ] };
+    key <AD11> { [    bracketleft,    braceleft, 	scaron,		Scaron  ] };
+    key <AD12> { [    bracketright,    braceright, 	dstroke, 	Dstroke ] };
+    key <AC04> { [    f,    F,	bracketleft             ] };
+    key <AC05> { [    g,    G,	bracketright            ] };
+    key <AC08> { [    k,    K, 	lstroke                 ] };
+    key <AC09> { [    l,    L,	Lstroke                 ] };
+    key <AC10> { [    semicolon,    colon,	ccaron,		Ccaron  ] };
+    key <AC11> { [    apostrophe,    quotedbl, 	cacute,		Cacute  ] };
+    key <LSGT> { [    less,    greater, 	bar			] };
+    key <AB01> { [    z,    Z,	y,		Y       ] };
+    key <AB04> { [    v,    V,	at                      ] };
+    key <AB05> { [    b,    B,	braceleft               ] };
+    key <AB06> { [    n,    N,	braceright              ] };
+    key <AB07> { [    m,    M,	section                 ] };
+    key <AB08> { [    comma,    less, 	semicolon               ] };
+    key <AB09> { [    period,    greater, 	colon			] };
+    key <AB10> { [    slash,    question, 	minus,    underscore  	] };
+    key <BKSL> { [    backslash,    bar,	zcaron,	    Zcaron      ] };
 
     include "level3(ralt_switch)"
 
